packages/block-library/src/video/editor.scss

```diff
@@ -56,3 +56,8 @@
 		padding: 0;
 	}
 }
+
+// remove top border from tools panel from video block inspector controls.
```

Comment:
Is there a way we can actually make the aspect ratio control part of the _settings group_, rather than add this CSS to override?